src/frontEnd/nav/initialNavState.ts

```diff
@@ -1,7 +1,7 @@
 import { urlFromString } from "../../utils/url/urlFromString";
 
 export const initialNavState = () => ({
-    isSideNavExpanded: true,
-    isSideNavOpen: true,
+    isSideNavExpanded: false,
+    isSideNavOpen: false,
```

Comment:
can we eliminate one of these? they are always in sync / the same thing, is that correct?

Reply:
so far, that is the case. I have not noticed / come accross a situation where they are, or need to be, different

Reply:
ok please search and determine if one can be eliminated and replaced with the other; if it can be, let's do it! no need to maintain both

Reply:
K. I searched and found no reason for the distinction. I think Mike built SideNav. I wanna consult with him to make sure I'm not missing anything,